pontoon/base/static/js/fluent_interface.js

```diff
@@ -23,6 +23,57 @@ var Pontoon = (function (my) {
       );
   }
 
+  function renderOriginalVariant(value, elements) {
+    return '<li>' +
+      '<span class="id">' +
+        value +
+      '</span>' +
+      '<span class="value">' +
+        Pontoon.fluent.serializePlaceables(elements, true) +
+      '</span>' +
+    '</li>';
+  }
+
+  function renderOriginalSimpleElement(simpleElements, title) {
+    if (simpleElements.length) {
+      var response = renderOriginalVariant(title, simpleElements);
+      simpleElements = [];
+      return response;
+    }
+
+    return '';
+  }
+
+  /*
+   * Render original string elements.
+   */
+  function renderOriginalElements(elements, title) {
+    var content = '';
+    var simpleElements = [];
+
+    elements.forEach(function (element, i, array) {
+      // Adjoining simple elements are concatenated and presented as a simple string
+      if (Pontoon.fluent.isSimpleElement(element)) {
+        simpleElements.push(element);
+
+        if (i === array.length - 1) {
+          content += renderOriginalSimpleElement(simpleElements, title);
+        }
+      }
+
```

Comment:
No empty line here, otherwise it looks like the `if` is finished, which is not the case.

Reply:
Agreed. OTOH it makes the code less readable. I like to separate blocks with an empty line, which is why I did it in other places as well.

So why don't we do this?

```js
// If condition is true
if (true) {
  a = 0;

// If condition is false
} else {
  a = 1;
}
```

Reply:
It's really a matter of personal preference. I like to have a clearly marked end of block, with the `}` alone on a new line. That's how I'm used to reading code, and thus what is easiest on me. I've always found `} else {` quite confusing because it kinda hides the end of the block. 

But overall, this is nitpicking and I can live with those newlines and stuff. 

On your example...

```javascript
// Do something.
if () {
    a();

// Do something else.
} else {
    b();
}
```

```javascript
// Do something.
if () {
    a();
}
// Do something else.
else {
    b();
}
```

```javascript
// Do something.
if () {
    a();
}

// Do something else.
else {
    b();
}
```
The second version to me is better because blocks are better identified. The third version hints me that the `else` block isn't related to the previous `if`. I use newlines to separate logical blocks of code, and there those two blocks are related, so they should be kept together. 

Hope I make sense, if you disagree it's fine! :)

Reply:
I see your point, but as you said - it's a matter of personal preference. :)

Since we decided not go beyond linters(:D) in code style reviews, I'll take the liberty of keeping this unchanged.